mentat/session.py

```diff
@@ -45,7 +45,7 @@ def __init__(
 
         cost_tracker = CostTracker()
 
-        code_context = CodeContext(stream, git_root, diff, pr_diff)
+        code_context = CodeContext(stream, git_root, diff, pr_diff, ignore_paths)
```

Comment:
I think this is supposed to be exclude_paths (although I could be wrong)

Reply:
Nah it's supposed to be `ignore_paths`. Should update the naming at some point tho